src/components/Assets/Assets.js

```diff
@@ -0,0 +1,246 @@
+import React, { Component } from "react";
+import AssetTable from "./AssetTable";
+import AssetMap from "./AssetMap";
+import AssetDetailsSection from "./AssetDetailsSection";
+import { FIELDS } from "./fieldConfig";
+import { getAllAssets } from "../WorkOrder/helpers";
+import { getAllAssetDetails, formatDataTitles } from "./helpers";
+
+import Autocomplete from "react-autocomplete";
+import changeCase from "change-case";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faMapMarkerAlt,
+  faSpinner,
+  faInfoCircle,
+} from "@fortawesome/free-solid-svg-icons";
+import {
+  Accordion,
+  AccordionItem,
+  AccordionItemTitle,
+  AccordionItemBody,
+} from "react-accessible-accordion";
+import "react-accessible-accordion/dist/fancy-example.css";
+
+class Assets extends Component {
```

Comment:
So we talked about this verbally, but would we need to implement different class components for the different asset types since they have different data? If so, we might want to check with @johnclary and @dianamartin to prioritize those asset types. My understanding is the assets are most important for Preventative Maintenance workflows. I know signals have regular PMs but I'm not sure about the others.

Reply:
@johnclary @dianamartin I'm curious about this as well. So far, I've found that, in Knack, work orders for school beacons, sensors, DMS, and cameras don't have a link from the Work Order Details page to the Asset Details page (using https://atd.knack.com/7-feb-2019--test-atd-data-tracker#work-orders/ to find records with these assets). They do have Asset Detail views for them, though (see below). 

https://atd.knack.com/7-feb-2019--test-atd-data-tracker#work-orders/dms-details/5c5c86a1ffdb0f62c74cb698/
https://atd.knack.com/7-feb-2019--test-atd-data-tracker#work-orders/school-flashers-zone-details/5c5c8748ffdb0f62c74d23c8/
https://atd.knack.com/7-feb-2019--test-atd-data-tracker#work-orders/camera-details/5c5c85cfffdb0f62c74c0a5f/
https://atd.knack.com/7-feb-2019--test-atd-data-tracker#work-orders/sensor-details/5c5c8d43ffdb0f62c750d7e3/